setup.cfg

```diff
@@ -6,6 +6,6 @@ max-line-length = 120
 # W503 raises a warning when there is a line break before a binary operator.
 # This is best practice according to PEP 8 and the rule should be ignored. 
 #
-# https:#www.flake8rules.com/rules/W503.html
-# https:#www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
+# https://www.flake8rules.com/rules/W503.html
+# https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
```

Comment:
This should be squashed into first commit.

Reply:
@findepi I was being lazy 😔 This is fixed now.